weather@mockturtl/files/weather@mockturtl/po/de.po

```diff
@@ -385,21 +420,66 @@ msgstr "Wettersymbole anzeigen"
 msgid "Display current temperature in panel"
 msgstr "Aktuelle Temperatur in der Leiste anzeigen"
 
+#. settings-schema.json->presentation-section->title
+msgid "Presentation"
+msgstr "Darstellung"
+
+#. settings-schema.json->verticalOrientation->description
+msgid "Show forecast vertically"
+msgstr "Vorhersage vertikal anzeigen"
+
+#. settings-schema.json->verticalOrientation->tooltip
+msgid "Displaying forecasts from top to bottom instead from left to right"
+msgstr "Vorhersage vertikal anstatt horizontal anzeigen"
+
+#. settings-schema.json->forecastColumns->units
+msgid "columns"
+msgstr "Spalten"
+
+#. settings-schema.json->forecastColumns->description
+msgid "Maximum number of columns for forecast"
+msgstr "Max. Anzahl der Spalten für die Vorhersage"
+
+#. settings-schema.json->forecastRows->units
+msgid "rows"
+msgstr "Reihen"
+
+#. settings-schema.json->forecastRows->description
+msgid "Maximum number of rows for forecast"
+msgstr "Max. Anzahl der Reihen für die Vorhersage"
+
+#. settings-schema.json->tempRussianStyle->description
+msgid "Show temperature Russian style"
+msgstr "Temperaturen „Russian Style“"
+
+#. settings-schema.json->units-section->title
+msgid "Units, Icons and Keys"
+msgstr "Einheiten, Icons & Schlüssel"
+
+#. settings-schema.json->info-section->title
+msgid "Help"
+msgstr "Hilfe"
+
+#. settings-schema.json->tempTextOverride->description
+msgid "Override label on panel"
+msgstr "Alternatives Label für die Leiste"
+
 #. settings-schema.json->showCommentInPanel->description
 msgid "Show the weather condition (e.g., \"Windy\", \"Clear\") in the panel"
-msgstr "Wetterbedingungen wie bspw. „Windig“ oder „Heiter“ in der Leiste anzeigen"
+msgstr ""
+"Wetterbedingungen wie bspw. „Windig“ oder „Heiter“ in der Leiste anzeigen"
```

Comment:
The text should be on line 469 and line 470 should be removed

Reply:
How can i change it? Or don't i need to do that? Did you you already change it? Thx for a hint.... 🙂